frontend/src/components/Vote/style.scss

```diff
@@ -0,0 +1,182 @@
+.votepic {
+  position: absolute;
+  height: 9em;
+  left: 23%;
+  top: 7%;
+}
+.votepictwo {
+  position: absolute;
+  height: 9em;
+  left: 68%;
+  top: 80%;
+}
+
+.votepicthree {
+  position: absolute;
+  height: 9em;
+  left: 23%;
+  top: 153%;
+}
+.house {
+  position: absolute;
+  height: 9em;
+  left: 65%;
+  top: 6.5%;
+}
+.house2 {
+  position: absolute;
+  height: 9em;
+  left: 25%;
+  top: 80%;
+}
+
+.house3 {
+  position: absolute;
+  height: 9em;
+  left: 65%;
+  top: 152%;
+}
+
+.line {
+  position: absolute;
+  height: 6em;
+  width: 100%;
+  left: 0%;
+  top: 65%;
+  transform: rotate(-180deg);
+}
+
+.line2 {
+  position: absolute;
+  height: 6em;
+  width: 100%;
+  left: 0%;
+  top: 140%;
+  transform: rotate(-360deg);
+}
+
+.line3 {
+  position: absolute;
+  height: 7em;
+  width: 100%;
+  left: 0%;
+  top: -3%;
+  transform: rotate(-360deg);
+}
+
+.line4 {
+  position: absolute;
+  height: 6em;
+  width: 100%;
+  left: 0%;
+  top: 70%;
+  transform: rotate(-360deg);
+}
+.line5 {
+  position: absolute;
+  height: 6em;
+  width: 100%;
+  left: 0%;
+  top: 135%;
+  transform: rotate(-180deg);
+  fill: red;
+}
+
+.center-items {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#fpro1 {
+  margin-top: 3vh;
+  width: 55vw;
+  height: 25vh;
+  margin-bottom: 7vh;
+}
+
+#fpro2 {
+  margin-top: 3vh;
+  width: 55vw;
+  height: 25vh;
+  margin-bottom: 7vh;
+}
+
+#fpro3 {
+  margin-top: 3vh;
+  width: 55vw;
+  height: 25vh;
+  margin-bottom: 7vh;
+}
+
+.proptitle {
+  display: flex;
+  margin-top: 8rem;
+
+  align-items: center;
+  justify-content: center;
+  color: whitesmoke;
+  text-shadow: 2px 2px black;
+  font-size: 1.6em;
+  font-weight: bolder;
+  text-decoration: underline 2px whitesmoke;
+  text-decoration-thickness: 3px;
+}
+
+.vote-button {
+  text-align: center;
+  margin: 0 auto;
+  margin-bottom: 14vh;
+  width: 20%;
+  font-size: 20px;
+  font-weight: bolder;
+  letter-spacing: 1px;
+  padding: 13px 20px 13px;
+  outline: 0;
+  border: 1px solid black;
+  cursor: pointer;
+  position: relative;
+  background-color: rgba(0, 0, 0, 0);
+  user-select: none;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+}
+
+.vote-button:after {
+  content: "";
+  background-color: #f8df4d;
+  width: 100%;
+  z-index: -1;
+  position: absolute;
+  height: 100%;
+  top: 7px;
+  left: 7px;
+  transition: 0.2s;
+}
+
+.vote-button:hover:after {
+  top: 0px;
+  left: 0px;
+}
+
+.pstyle {
+  width: 100%;
```

Comment:
Useless in most cases, that's the default width

Reply:
fix